@here/harp-datasource-protocol/lib/Theme.ts

```diff
@@ -966,6 +966,12 @@ export interface DirectionalLight extends BaseLight {
     intensity: number;
     direction: Vector3Like;
     castShadow?: boolean;
+
+    /**
+     * Number between 0 and 1 which controls how strong the shadow is on flat surfaces.
+     * 0 means there is no shadow and 1 means the shadow appears as pure black.
+     */
+    shadowIntensity?: number;
```

Comment:
Shouldn't this be in the material/technique?

Reply:
Why would the designer want to customize it per material?

Reply:
Who knows. I don't see why we should restrict them.

Reply:
Flexibility has its cost, we have to provide long term maintenance support, and removing it is difficult. Without a clear use case, I would prefer something simple that works. It would be easy to remove this later with minimal cost if there is a use case where a per material shadow is required.

I am also not so keen on mixing shadow properties with materials, to me it belongs with the light definition.

Maybe a good compromise, and one which would actually make sense, I remove the shadowIntensity and instead multiply the material color with the ambient light, that way, the ambient would actually make some sense and it would remove this parameter.

Reply:
I don't see why a per technique property is harder to maintain.
It's fine for me if we extend functionality later b/c it's to much effort to do it now but in this case we would have to deprecate the shadow intensity in the light source b/c some people might use it.
Or are you proposing to have both in the the future?

Not fully understand your last comment. How is the ambient light related to the shadows.

Maybe a third alternative to go forward is to just use the 0.46 intensity (where is this number coming from btw.?) and don't make it configurable for now.

Reply:
Every time a designer wants to style some thing with "fill" or "standard", they need to also copy across the shadow intensity.

Anyway, I still don't see how it is related to a material property.

Regarding the ambient light, we could say that the pixel color in shadow is material.color * ambientLight, because the material is lit just from the ambient light (and not the directional light).

Reply:
Yeah, it probably doesn't need to be configured, and when it does, we can find the best solution then, I will remove for now.